docs/developers-guide/driver-changelog.md

```diff
@@ -6,6 +6,12 @@ title: Driver interface changelog
 
 ## Metabase 0.50.0
 
+- The multimethod `metabase.driver/describe-fields` has been added. The method needs to be
+  implemented if the database supports the `:fast-sync-fields` feature.
+
+- The multimethod `metabase.driver.sql-jdbc.sync.describe-table/describe-fields-sql` has been added. The method needs
+  to be implemented if you want to use the default JDBC implementation of `metabase.driver/describe-fields`.
```

Comment:
should be 49

Reply:
I've optimistically changed it to 49.1 in 137c1404a72ed9b0c3d1ad20e1a21778c73ab8db